4.1/buster/hooks/post_push

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
I don't think these hooks are necessary here. There just has to be one because Docker Hub builder wants to always tag the last build as `latest`.